README.md

```diff
@@ -12,4 +12,20 @@ Use Node version 14 or 16.
 ```sh
 npm install
 npm test # This performs tests and checks test coverage
-```
\ No newline at end of file
+```
+
+## Contributing
+
+Style guide
+
+- 120 hard limit line length
+- NatSpec
+  - Align whitespace for tags
```

Comment:
Extra indentation looks good in code but they throw off doc generation scripts. They create whitespace gaps in text and it looks ugly. Because of that I decided to not to use indentation in code.